packages/zoe/test/unitTests/test-zoe.js

```diff
@@ -1,9 +1,7 @@
-/* global __dirname */
+/* global __dirname test */
```

Comment:
This uses `test` as a global, but it doesn't get one from anywhere.

Reply:
It comes from prepare-test-env.js

Reply:
I'm looking at this while trying out something similar... why a global?